go.mod

```diff
@@ -26,6 +26,8 @@ require (
 	github.com/giantswarm/helmclient v0.2.0
 	github.com/giantswarm/ipam v0.2.0
 	github.com/giantswarm/k8sclient v0.2.0
+	github.com/giantswarm/k8sclient/v2 v2.0.0
+	github.com/giantswarm/k8sclient/v3 v3.0.0
```

Comment:
I need to use two different versions of the same library: we need to different versions until all the CRDs are graduated.